index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <header id="header">
+      <img id="header-img" src="https://assets-global.website-files.com/5dbb30f00775d4c32191a4df/5e30adbb408180632f50a480_horizontal-logo-white-p-500.png
+      ">
```

Comment:
- [ ] If you use the `img` tag, you should use an `alt` attribute for the `img` tag. That way, you will be helping those who use screen readers to navigate your site easily. DO THAT NOW!!!